taskcluster/kinds/build/kind.yml

```diff
@@ -7,9 +7,24 @@ loader: taskgraph.loader.transform:loader
 transforms:
     - ffios_taskgraph.transforms.bitrise
     - taskgraph.transforms.task
-
+task-defaults:
+    worker-type: bitrise
+    bitrise:
+        artifact_prefix: public
 
 tasks:
+    build-firefox-debug:
```

Comment:
nit: the `kind` implicitly gets added to the name here, so the task label will actually be `build-build-firefox-debug`. You can remove the `build` here to fix it.